content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -315,7 +316,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `AffinityInAnnotations`(*deprecated*): Enable setting [Pod affinity or anti-affinitys](/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity).
 - `AllowExtTrafficLocalEndpoints`: Enable a service to route external requests to node local endpoints.
 - `APIListChunking`: Enable the API clients to retrieve (`LIST` or `GET`) resources from API server in chunks.
-- `APIPriorityAndFairness`: Enable managing request concurrency with prioritization and fairness at each server.
+- `APIPriorityAndFairness`: Enable managing request concurrency with prioritization and fairness at each server. (Renamed from `RequestManagement`)
```

Comment:
@makocchi-git , Looks good 😃 . One nit:
... fairness at each server. This feature gate is formerly known as `RequestManagement`.

Reply:
@kbhawkey Thank you for your advice👍